administrator/components/com_fields/src/Model/FieldsModel.php

```diff
@@ -190,7 +190,7 @@ protected function getListQuery()
 			{
 				$access = (int) $access;
 				$query->where($db->quoteName('a.access') . ' = :access')
-					->bind(':access', $access, ParameterType::INTEGER);
+				->bind(':access', $access, ParameterType::INTEGER);
```

Comment:
```suggestion
				->bind(':access', $access, ParameterType::INTEGER);
```
Same as my 2 previous review comments.

You can find the results of the PHP CS check by following the "details" link of the drone check. Then you select the failed test (PHPCS) at the left hand side and see the report at the right hand side: https://ci.joomla.org/joomla/joomla-cms/49101/1/6